k8s_services_srv.py

```diff
@@ -135,20 +135,38 @@ def del_dynamo_cluster_backend(cluster, endpoint, table):
         return True
 
 
-def list_k8s_services(namespace, label_selector):
-    # Get the k8s service with specific labels
+def clean_dynamo_cluster_backend(table):
+    # Cleaning endpoints with expired TTL
+    logging.info("Cleaning up the DB backend")
+    response = table.scan()
+    items = response['Items']
+    for item in items:
```

Comment:
for item in response['Items']

La création d'une variable est inutile dans ce cas la

Reply:
Faut bien utiliser toute cette mémoire 👍 
Changé !